src/functions/parse_cookie_header.legacy.php

```diff
@@ -8,9 +8,10 @@
 use function Laminas\Diactoros\parseCookieHeader as laminas_parseCookieHeader;
```

Comment:
Maybe we just drop this as we already removed support for the old zendframework implementation by adding `conflict` to `composer.json`?

Reply:
We do declare these symbols: we can drop them, but not here :thinking: